src/course/en/articles.json

```
@@ -37,24 +39,38 @@
             "_banks": {
                 "_isEnabled": true,
                 "_split": "2,2",
-                "_randomisation": true
+                "_randomisation": false
             },
             "_questions": {
                 "_resetType": "hard",
                 "_canShowFeedback": false,
                 "_canShowMarking": false,
                 "_canShowModelAnswer": false
             },
-            "_isResetOnRevisit": true,
+            "_isResetOnRevisit": false,
```

please revert back to old settings
```suggestion
            "_isResetOnRevisit": true,
```